.gitignore

```diff
@@ -80,4 +80,7 @@ typings/
 .next
 
 # document generation directory
-.docz
\ No newline at end of file
+.docz
+
```

Comment:
I don't have the latest changes from your branch but at one point it seemed like the Windows bundle of the test app wasn't being ignored by  git

Reply:
Yeah I moved the bundle location to be under dist to get it ignored for right now